nemo/collections/tts/torch/data.py

```diff
@@ -361,6 +363,23 @@ def add_pitch(self, **kwargs):
         self.pitch_std = kwargs.pop("pitch_std", None)
         self.pitch_norm = kwargs.pop("pitch_norm", False)
 
+    # saving voiced_mask and p_voiced with pitch
+    def add_voiced_mask(self, **kwargs):
+        self.voiced_mask_folder = kwargs.pop('voiced_mask_folder', None)
```

Comment:
Can you docstring your new kwargs `voiced_mask_folder` and `p_voiced_folder`.

Reply:
added docstring